lib/tasks/proposals.rake

```diff
@@ -1,7 +1,9 @@
 namespace :delete_proposals do
   desc 'Delete proposals that are older than specified date old'
   task expired_proposals: :environment do
-    count = CollectionDraftProposal.delete_all("proposal_status = 'in_work' AND updated_at < '#{30.days.ago}'") if Rails.configuration.proposal_mode
-    Rails.logger.info("Automated check for proposals that are 'in_work' and 30 days old deleted #{count || 0} proposals.")
+    if Rails.configuration.proposal_mode
```

Comment:
Is rubocop complaining that you should use a guard clause? I either think this way _or_ the guard clause would be fine, was just thinking we should change it if rubocop is complaining